### DISCUSSION TITLE
230710 1차 데모데이 피드백 검토 (16:00~17:00)

### DISCUSSION THREAD

**Discussion by seokhwan-an** [woowacourse-teams/2023-shook#35] [회의]
### 회의 안건
1차 데모데이 때 받은 피드백을 검토한다.

**Comment by Cyma-s** [comment]
## 피드백 검토
- 깃 브랜치 전략을 말씀 주셨는데, release 브랜치가 필요하면 추가한다고 했잖아. 배포한 상황이 없을 텐데, 조금 더 간소한 브랜치 전략을 가져가는 것은 어떨지? -> 그대로 Git Flow 변형 버전으로 진행한다.
- 서비스 플로우를 더 구체적으로 말해주시면 좋을 듯. 킬링파트가 어떻게 정해지는지. -> 이미 플로우는 나와 있는 상태
    - 어뷰징 관련 대처? -> 계정당 노래 1개마다 1회 투표. 로그인 기능 구현 시 고려. 어느 정도 정해지면 추후 고려해보자.
    - 아무도 킬링파트를 지정하지 않은 경우? -> 킬링파트가 없는 경우듣기만 하는 유저들에게는 서비스 자체의 의미가 사라질 수 있다. 더미데이터를 어느 정도 추가할 필요가 있다. 크루들에게 이벤트 형식으로 데이터를 받는 것도 좋을 것 같다.
- 궁극적으로 음악 데이터를 어디서 가져올 것인가? -> 일단은 유튜브 API를 사용한다.
- 내 취향 아카이브 vs 노래에 대한 킬링파트에 대한 정보가 우선인지
    - 현재 내 취향의 노래를 저장할 수 있는 기능은 존재하지 않는다. 그렇지만 우리가 정한 페르소나에서는 충분히 필요로 할 수 있을 법한 기능.
    - 킬링파트에 대한 정보를 얻을 수 있는 것을 중심으로 하되, 내 취향을 모아두는 기능(가치)이 필요할 수도 있다.
- “팀의 의견은 모두의 의견입니다” 에 대한 구체적인 방안을 강구하는 게 좋을 것 같다. 최소한의 무엇을 하겠다라는 규칙을 정해야할 것 같다. 피드백할 수 있는 구체적인 것!
    - 결정된 사항에 찝찝한 사람이 있을 수 있다. -> 중대 사안에 대해서 기명 설문(의견에 동의하는 정도에 따라 1~5점 주기 + 이유)을 실시해서, 다른 의견이 있는지 확인한다.
